src/components/Synthetics/MarketStats/MarketStats.tsx

```diff
@@ -110,13 +111,15 @@ export function MarketStats(p: Props) {
             />
           }
         />
+
         <CardRow
           label={t`Wallet`}
-          value={
-            marketBalance && marketBalanceUsd
-              ? formatTokenAmountWithUsd(marketBalance, marketBalanceUsd, "GM", marketToken?.decimals)
-              : "..."
-          }
+          value={formatTokenAmountWithUsd(
+            marketBalance || BigNumber.from(0),
+            marketBalanceUsd || BigNumber.from(0),
+            "GM",
+            marketToken?.decimals || 18
```

Comment:
don't you prefer using `??` for such logic? I think it's more appropriate for non-logic operations.

Reply:
Updated!